ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/monitor-sessions.md

```diff
@@ -94,6 +94,24 @@ interface Management1
 | -------- | ------ |
 | Destinations | - |
 
+##### myMonitoringSession4
```

Comment:
In line 89 the per source ACL is still printed even though it is ignored in the config.

Reply:
Shall I print access_group set at session level instead of empty cell?

Reply:
> In line 89 the per source ACL is still printed even though it is ignored in the config.

Isn't line 89 OK? It covers myMonitoringSession3 which has no ACL-based collisions. Is issue more on line 71 (with myMonitoringSession2)?

Reply:
Yes, issue is on line 71. Shall I print this from 68 to 71. As ip access group ipv4ACL is present on myMonitoringSession2.

| Ethernet3, Ethernet5 | rx | ip | ipv4ACL | - |
| Ethernet10-15 | rx | ip | ipv4ACL | - |
| Ethernet12 | rx | ip | ipv4ACL | - |
| Ethernet18 | tx | ip | ipv4ACL | 100 |

Reply:
No you should just print the - if we are ignoring the ACL like we do in the EOS config.

Reply:
Fixed.